src/components/SelectionList/BaseSelectionList.js

```diff
@@ -290,121 +301,114 @@ function BaseSelectionList({
-                                    viewabilityConfig={{viewAreaCoveragePercentThreshold: 95}}
-                                    testID="selection-list"
-                                    onLayout={() => {
-                                        if (!firstLayoutRef.current) {
-                                            return;
-                                        }
-                                        scrollToIndex(focusedIndex, false);
-                                        firstLayoutRef.current = false;
-                                    }}
-                                />
-                            </>
-                        )}
-                        {showConfirmButton && (
-                            <FixedFooter>
-                                <Button
-                                    success
-                                    style={[styles.w100]}
-                                    text={confirmButtonText || translate('common.confirm')}
-                                    onPress={onConfirm}
-                                    pressOnEnter
-                                    enterKeyEventListenerPriority={1}
-                                />
-                            </FixedFooter>
-                        )}
-                    </View>
-                )}
-            </SafeAreaConsumer>
-        </ArrowKeyFocusManager>
+                                    />
+                                    <View style={[styles.flex1]}>
+                                        <Text style={[styles.textStrong, styles.ph3]}>{translate('workspace.people.selectAll')}</Text>
+                                    </View>
+                                </PressableWithFeedback>
+                            )}
+                            <SectionList
+                                ref={listRef}
+                                sections={sections}
+                                renderSectionHeader={renderSectionHeader}
+                                renderItem={renderItem}
+                                getItemLayout={getItemLayout}
+                                onScroll={onScroll}
+                                onScrollBeginDrag={onScrollBeginDrag}
+                                keyExtractor={(item) => item.keyForList}
+                                extraData={focusedIndex}
+                                indicatorStyle="white"
+                                keyboardShouldPersistTaps="always"
+                                showsVerticalScrollIndicator={showScrollIndicator}
+                                initialNumToRender={12}
+                                maxToRenderPerBatch={5}
+                                windowSize={5}
+                                viewabilityConfig={{viewAreaCoveragePercentThreshold: 95}}
+                                testID="selection-list"
+                                style={[styles.flexGrow0]}
+                                onLayout={() => {
```

Comment:
Can we extract it into a callback like a handle function?

Reply:
Sure thing!